config/puma.rb

```diff
@@ -1,25 +1,37 @@
-# https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server
+# Puma can serve each request in a thread from an internal thread pool.
+# The `threads` method setting takes two numbers: a minimum and maximum.
+# Any libraries that use thread pools should be configured to match
+# the maximum value specified for Puma. Default is set to 5 threads for minimum
+# and maximum; this matches the default thread size of Active Record.
+#
+threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
+threads threads_count, threads_count
 
-# The environment variable WEB_CONCURRENCY may be set to a default value based
-# on dyno size. To manually configure this value use heroku config:set
-# WEB_CONCURRENCY.
+# Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-# Increasing the number of workers will increase the amount of resting memory
-# your dynos use. Increasing the number of threads will increase the amount of
-# potential bloat added to your dynos when they are responding to heavy
-# requests.
+port        ENV.fetch("PORT") { 3000 }
```

Comment:
So many empty spaces 👾 👾 . Seems odd 🤔

Reply:
Yes, I cannot endorse this particular generated line's styling.